config_defaults_inc.php

```diff
@@ -1214,6 +1214,16 @@
  */
 $g_private_news_threshold = DEVELOPER;
 
+############
+# Markdown #
+############
+
+/**
+ * Enables markdown support - once enabled all issue titles, text, and notes are formatted based on
+ * markdown syntax.  Default to OFF since it is experimental feature for now.
+ */
+$g_markdown_enabled = OFF;
```

Comment:
>  this configuration is no different than others like bug_link_tag and bugnote_link_tag.

Thats what i mean. See:
![seleccion_075](https://cloud.githubusercontent.com/assets/14123811/19628675/88438236-9963-11e6-9c88-325e8c1e6650.png)

Having a global config when this can be done cleanly in the already dedicated functionality, seems counterintuitive to me